src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/TagParser.java

```diff
@@ -106,7 +106,7 @@ public void parse(@Nonnull String json, @Nonnull BiConsumer<Set<Material>, Set<T
                 JsonArray values = child.getAsJsonArray();
 
                 for (JsonElement element : values) {
-                    if (element instanceof JsonPrimitive && ((JsonPrimitive) element).isString()) {
+                    if (element instanceof JsonPrimitive jsonPrimitive && jsonPrimitive.isString()) {
```

Comment:
We do not always have to be so literal about it.
```suggestion
                    if (element instanceof JsonPrimitive primitive && primitive.isString()) {
```